AirQualityMonitoring/schema.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
I have reviewed first detail by detail and then globally. My question, how could we ensure a quick and simple interoperability if we do not define what are the objects' properties? There is not a problem with defining them  as objects and defining all the possible values (or extending later). But how could you prepare on the other end the reading of the values? In fact, what you are doing is creating a not documented standard (under the need for being very flexible). 
Let's comment this

Reply:
Hey Alberto, I have made some changes with respect to the schema.json document I believe it should have all the properties required now. If there are modifications required in the payload files please let me know I will modify them as well.

Reply:
With regard to the reading of the values we have a different file called the Data Descriptor in which the detailed description of each and every property is mentioned.You can check out Example 1 in this [IUDX Vocabulary](https://voc.iudx.org.in/EnvAQM). This contains the detailed information  about each of the properties in the data model.